### DISCUSSION TITLE
How to solve Quota api.dw.util.collectionSize exceptions

### DISCUSSION THREAD

**Discussion by eigomez** [SalesforceCommerceCloud/sfcc-ci#456] [Team Posts]
Hi, we are having this issue right now, any suggestions will be appreciated.

Limit for quota 'api.dw.util.collectionSize' exceeded. Limit is 20000, actual is 20001.

Thanks.

**Comment by pmulagandla** [comment]
Hi team, To solve this issue you have to check for the current count of objects and if the limit has been reached create a new collection which will store the additional objects.
Thank you.

**Comment by eigomez** [comment]
Thanks, @pmulagandla should be this created in business manager?

**Comment by DavidKingII** [comment]
Look at you sitemap as well.

________________________________
